tests/templates/test_subroutines/test_adder.py

```diff
@@ -118,6 +118,7 @@ def circuit(x):
         if mod is None:
             mod = 2 ** len(x_wires)
 
+        # pylint: disable=bad-reversed-sequence
```

Comment:
Why this is needed?

Reply:
I wish I knew - my pylint install insisted 🤷‍♀️  My best guess is that it somehow isn't sure what the type of the output of circuit(x) is, so it can't be sure the argument has a `__reversible__` property and assumes the worst. But the output in this instance is an array, which is definitely reversible.